### PR TITLE
Mention the preferred version of GCC on Windows

### PR DESCRIPTION
**Pull request by pdbain-ibm**
Related to #727 and #922 

Signed-off-by: Peter Bain <peter_bain@ca.ibm.com>

### CODE REVIEW

**Comment by charliegracie** [comment]
I know in the past the IBM SDK used a different version on mingw (gcc) than the one listed here.  Do you know if the version listed here generates code that performs as well as the older versions of mingw?

**Comment by pshipton** [comment]
@charliegracie see #727 for the performance results

**Comment by pdbain-ibm** [comment]
This is ready for review & merge.

File: buildenv/Build_Instructions_V9.md [hunk 1]
```diff
@@ -311,7 +311,7 @@ The following instructions guide you through the process of building a Windows O
 :ledger:
 You must install a number of software dependencies to create a suitable build environment on your system:
 
-- [Cygwin](https://cygwin.com/install.html), which provides a Unix-style command line interface. Install all packages in the `Devel` category. In the `Archive` category, install the packages `zip` and `unzip`. Install any further package dependencies that are identified by the installer. More information about using Cygwin can be found [here](https://cygwin.com/docs.html).
+- [Cygwin](https://cygwin.com/install.html), which provides a Unix-style command line interface. Install all packages in the `Devel` category. In the `Archive` category, install the packages `zip` and `unzip`. Install any further package dependencies that are identified by the installer. More information about using Cygwin can be found [here](https://cygwin.com/docs.html). OpenJ9 is tested with version 6.4.0 of GCC, provided by default with current Cygwin installations.  Other versions are expected to work also.
```
Comment (pshipton) [inline buildenv/Build_Instructions_V9.md]: Should this way mingw instead of GCC?

Comment (pshipton) [reply buildenv/Build_Instructions_V9.md]: Or mingw/gcc

**Comment by pdbain-ibm** [comment]
Thanks for the comment.
My original wording was awkward, so I reworded it and added more explanation.

**Comment by pshipton** [comment]
@SueChaplain fyi

**Pull request merged**